site/frontend/src/pages/compare/compile/table/benchmark-detail.vue

```diff
@@ -149,88 +179,209 @@ const cargoProfile = computed((): CargoProfileMetadata => {
-            <td>Codegen units</td>
-            <td>{{ cargoProfile.codegen_units }}</td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-    <div>
-      <div class="title">
-        <div class="bold">{{ getGraphTitle() }}</div>
-        <div style="font-size: 0.8em">
-          Each plotted value is relative to its previous commit
+  <div>
+    <div class="columns">
+      <div class="rows grow">
+        <div>
+          <div class="title info bold">Benchmark info</div>
+          <table>
+            <tbody>
+              <tr>
+                <td>Benchmark</td>
+                <td>{{ testCase.benchmark }}</td>
+              </tr>
+              <tr>
+                <td>Profile</td>
+                <td>{{ testCase.profile }}</td>
+              </tr>
+              <tr>
+                <td>Scenario</td>
+                <td>{{ testCase.scenario }}</td>
+              </tr>
+              <tr>
+                <td>Category</td>
+                <td>{{ testCase.category }}</td>
+              </tr>
+              <tr v-if="(metadata?.binary ?? null) !== null">
+                <td>Artifact</td>
+                <td>{{ metadata.binary ? "binary" : "library" }}</td>
+              </tr>
+              <tr v-if="(metadata?.iterations ?? null) !== null">
+                <td>
+                  Iterations
+                  <Tooltip> How many times is the benchmark executed? </Tooltip>
+                </td>
+                <td>{{ metadata.iterations }}</td>
+              </tr>
+              <tr v-if="(cargoProfile?.lto ?? null) !== null">
+                <td>LTO</td>
+                <td>{{ cargoProfile.lto }}</td>
+              </tr>
+              <tr v-if="(cargoProfile?.debug ?? null) !== null">
+                <td>Debuginfo</td>
+                <td>{{ cargoProfile.debug }}</td>
+              </tr>
+              <tr v-if="(cargoProfile?.codegen_units ?? null) !== null">
+                <td>Codegen units</td>
+                <td>{{ cargoProfile.codegen_units }}</td>
+              </tr>
+            </tbody>
+          </table>
         </div>
-        <div style="font-size: 0.8em">
-          The shaded region shows values that are more recent than the
-          benchmarked commit
+        <div class="links">
+          <div class="title bold">Links</div>
+          <ul>
+            <li>
+              <a
+                :href="detailedQueryLink(props.artifact, props.baseArtifact)"
+                target="_blank"
+              >
+                Detailed results
+              </a>
+            </li>
+            <li>
+              <a
+                :href="graphLink(props.artifact, props.metric, props.testCase)"
+                target="_blank"
+              >
+                History graph
+              </a>
+            </li>
+            <li>
+              <a :href="detailedQueryLink(props.baseArtifact)" target="_blank">
+                Rustc self-profile: baseline commit
```

Comment:
I feel like this could also be "Detailed results: baseline commit", and dito for the next line, to indicate the parallel to the other "Detailed results" page.